README.md

```diff
@@ -70,6 +70,12 @@ and examples at [eodagmbh.github.io/py-maplibregl/examples/](https://eodagmbh.gi
 - [MapBlockly](https://mapblockly.github.io/) - MapBlockly is a simple and fun way to learn and build Map with Blockly using MapLibre.
 - [MapInventor](https://mapinventor.github.io/) - MapInventor is the visual language built on top of MapBlockly.
 
+## Sprites
```

Comment:
It's probably worth adding ekastic's spritezero, and maybe update the sprite spec documentation with tool to create sorites for maolibre.